package.json

```diff
@@ -184,6 +184,7 @@
     "statsig-react-native": "^4.9.2",
     "svgs": "^4.2.0",
     "tslib": "^2.4.1",
+    "use-debounce": "^9.0.3",
```

Comment:
Since we already use `debounce` from `lodash`, could we avoid this new dependency?

Reply:
I tried using Lodash and couldn't quite get to it to work. `use-debounce` is a pretty small library, plays with React well, and we also need the functionality in #3613. It would be nice to keep if the build size isn't increased much.

Reply:
yes it's not so much the build size, but trying to keep dependencies under control, since we're already struggling a bit with this.
Happy to help here.

Reply:
Got it sorted in 9f1d1e1367e7eafb7f2d7714c5993f0e67676d29 with help from @dievazqu!